docs/sample_inventories/rbac_sso_c3.yml

```diff
@@ -0,0 +1,108 @@
+---
+### RBAC SSL - Multiple KSQL Clusters
```

Comment:
looks like title and description are not updated, we have one more sample inventory `RBAC SSL - Multiple KSQL Clusters`

Reply:
thanks for pointing out, updated it